### PR TITLE
Bugfix/all 4863 doge getblock

### PR DESCRIPTION
**Pull request by rostislavjadavan**
# Description

Doge `getBlock` method has different type of `verbosity` param then other UTXO based chains.
See:
https://github.com/dogecoin/dogecoin/blob/3a29ba6d497cd1d0a32ecb039da0d35ea43c9c85/src/rpc/blockchain.cpp#L741

Fixes # (issue)

- ALL-4863

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Hathoriel** [comment]
upgrade package json and changelog

**Review by Hathoriel — CHANGES_REQUESTED** [review]

File: src/service/rpc/utxo/DogeRpc.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { GenericRpc } from '../generic'
+import { AbstractDogeRpc } from './AbstractDogeRpc'
+
+@Service({
+  factory: (data: { id: string }) => {
+    return new DogeRpc(data.id)
+  },
+  transient: true,
+})
+export class DogeRpc extends AbstractDogeRpc implements DogeRpcSuite {
```
[the remark this answers is not in the document]
Comment (rostislavjadavan) [reply src/service/rpc/utxo/DogeRpc.ts:14]: Could you be more specific, what is not needed?

[the remark this answers is not in the document]
Comment (Hathoriel) [reply src/service/rpc/utxo/DogeRpc.ts:14]: you are not using it. YOu are using DogeLoadBalancedRpc

[the remark this answers is not in the document]
Comment (rostislavjadavan) [reply src/service/rpc/utxo/DogeRpc.ts]: Right! Removed :)

**Review by Hathoriel — APPROVED** [review]

**Pull request merged**